modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/enrich/kinesis/Sink.scala

```diff
@@ -39,6 +39,9 @@ import com.snowplowanalytics.snowplow.enrich.common.fs2.config.io.Output
 
 object Sink {
 
+  // PutRecords request can be up to 5 MiB, including partition keys
+  val putRecordsSizeLimit = 5 * 1024 * 1024
```

Comment:
There are a few places in our snowplow estate where a config option has a _theoretical_ maximum, but we set the value slightly lower just in case there is an edge case we don't know about.

- [Here in enrich-pubsub](https://github.com/snowplow/enrich/blob/3.3.0/config/config.pubsub.extended.hocon#L56-L59)
- [Here in collector](https://github.com/snowplow/stream-collector/blob/master/kinesis/src/main/resources/application.conf#L19)

I'm sure there are other places too.  By putting it in `application.conf` it means we can override it in an emergency, just in case we discover an edge case in production.